basic/src/test/java/org/jboss/hal/testsuite/test/configuration/jgroups/JGroupsOperations.java

```diff
@@ -1,90 +1,39 @@
     }
 
-    public String getTransport() {
-        return transport;
-    }
-
-    public void setTransport(String transport) {
-        this.transport = transport;
-    }
-
-    private String getBaseDmrPath() {
-        return baseDrmPath;
-    }
-
-    public void setBaseDrmPath(String baseDrmPath) {
-        this.baseDrmPath = baseDrmPath;
-    }
-
-    private String getTransportDrmPath() {
-        return getBaseDmrPath() + "transport=" + transport + "/";
-    }
-
-    private String getProtocolDrmPath(String name) { //TODO: verify
-        return getBaseDmrPath() + "protocol=" + name + "/";
-    }
-
-    /**/
-
-    private boolean addProperty(String path, String name, String value) {
-        String command = path + "property=" + name + ":add(value=" + value + ")";
-        return client.executeForSuccess(command);
-    }
-
-    private void removeProperty(String path, String name) {
-        String command = path + "property=" + name + ":remove";
-        client.executeCommand(command);
-    }
-
-    public boolean verifyProperty(String path, String name, String value) {
-        String command = path + ":read-resource";
-        return client.executeForResponse(command)
-                .get("result")
-                .get("properties")
-                .toJSONString(false)
-                .contains("\"" + name + "\" : \"" + value + "\"");
-    }
-
-    /*transport properties*/
-
-    public boolean addTransportProperty(String name, String value) {
-        return addProperty(getTransportDrmPath(), name, value);
-    }
-
-    public void removeTransportProperty(String name, String value) {
-        removeProperty(getTransportDrmPath(), name);
-    }
-
-    public boolean verifyTransportProperty(String name, String value) {
-        return verifyProperty(getTransportDrmPath(), name, value);
-    }
-
-    /*protocol properties*/
-
-    public boolean addProtocolProperty(String protocol, String name, String value) {
-        return addProperty(getProtocolDrmPath(protocol), name, value);
+    public boolean addProperty(Address address, String name, String value) throws IOException {
+        ModelNodeResult result = operations.add(address.and("property", name), Values.of("value", value));
+        return result.isSuccess();
     }
 
-    public boolean verifyProtocolProperty(String protocol, String name, String value) {
-        return verifyProperty(getProtocolDrmPath(protocol), name, value);
+    public void removeProperty(Address address, String name) throws IOException, OperationException {
+        try {
+            operations.removeIfExists(address.and("property", name));
+        } catch (OperationException ignored) {
+            //operation fails if resource does not exists
+        }
     }
 
-    public void removeProtocolProperty(String protocol, String name) {
-        removeProperty(getProtocolDrmPath(protocol), name);
+    public boolean verifyProperty(Address address, String name) throws IOException {
```

Comment:
I suggest to change method name to something more descriptive like `propertyExists` or stuff like that since you doesn't 'verify' property existence inside this method.